keras_cv/layers/preprocessing/with_keypoints_test.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+from absl.testing import parameterized
+
+from keras_cv.layers import preprocessing
+
+# Imports from with_labels the list of augmentation that should perform a No-Op.
+from keras_cv.layers.preprocessing.with_labels_test import TEST_CONFIGURATIONS
+
+# List here the layers that are expected to modify keypoints with
+# their specific parameters.
+GEOMETRIC_TEST_CONFIGURATIONS = []
+
+
+class WithKeypointsTest(tf.test.TestCase, parameterized.TestCase):
+    @parameterized.named_parameters(
+        *TEST_CONFIGURATIONS,
+        *GEOMETRIC_TEST_CONFIGURATIONS,
+        ("CutMix", preprocessing.CutMix, {}),
```

Comment:
does cutmix work with keypoints?

Reply:
`CutMix` should do a No-Op on keypoint/bounding boxes no ? The Idea is here to ensure that a No-Op has been added to `CutMix`. It is the same why I import `TEST_CONFIGURATION` from `with_labels.py`, to be sure all of these transformation do have no-op operation implemented.